src/tests/test_dte_data_models.py

```diff
@@ -1645,24 +1793,26 @@ def test_validate_referencias_codigo_ref_is_consistent_with_tipo_dte(self) -> No
 
         expected_validation_errors = [
             {
-                'loc': ('__root__',),
-                'msg': "'codigo_ref' is mandatory for this 'tipo_dte':"
-                " 'tipo_dte' == <TipoDte.NOTA_CREDITO_ELECTRONICA: 61>,"
-                " 'Referencia' number 1.",
+                'loc': (),
+                'msg': "Value error, 'codigo_ref' is mandatory for this 'tipo_dte': 'tipo_dte' == "
+                "<TipoDte.NOTA_CREDITO_ELECTRONICA: 61>, 'Referencia' number 1.",
```

Comment:
Reformat to reduce diff size.

```suggestion
                'msg': "Value error, 'codigo_ref' is mandatory for this 'tipo_dte':"
                " 'tipo_dte' == <TipoDte.NOTA_CREDITO_ELECTRONICA: 61>,"
                " 'Referencia' number 1.",
```